Sources/Host/POSIX/ProcessSpawner.cpp

```diff
@@ -404,6 +406,7 @@ ErrorCode ProcessSpawner::run(std::function<bool()> preExecAction) {
       environment.push_back(nullptr);
 
       if (!preExecAction()) {
+        DS2LOG(Error, "pre exec action failed, cannot execute inferior");
```

Comment:
s/, cannot execute inferior//

This will be logged higher up in the stack IIRC (if it isn't we should add a log there too).